### ISSUE TITLE
Tag v3.0.0 is broken

### ISSUE THREAD

**Issue by sobolevnrm** [Electrostatics/apbs#89]
**Describe the bug**

https://github.com/Electrostatics/apbs/releases/tag/v3.0.0 has a submodule that is missing its URL.

**To Reproduce**
The following Dockerfile contents are based on the current install instructions:

```
FROM ubuntu
WORKDIR /root

# Install dependencies
RUN apt update && apt upgrade
RUN apt install -y git

# Clone repository
RUN git clone https://github.com/Electrostatics/apbs.git apbs
RUN cd apbs && git checkout v3.0.0
RUN cd apbs && git submodule init && git submodule update
```

When building this Dockerfile, the following error is returned on the last step:
```
fatal: No url found for submodule path 'externals/bem' in .gitmodules
```

**Expected behavior**

The last stable tag/release should work with the instructions at https://apbs.readthedocs.io/en/latest/getting/source.html.

**Comment by intendo** [comment]
This apbs repo for the v3.0.0 tag has never worked for me. I have always had to go back to the original repo `apbs-pdb2pqr` and checkout the `darren/valgrind` branch to build apbs and use it.

The docs work for the latest version of apbs in master, which I think we should make a new release to to get a stable tag/release.

**Comment by sobolevnrm** [comment]
Can we make a minimal increment on v3.0.0 that works?

________________________________

**Comment by intendo** [comment]
We have 2 conflicting problems.

A. To include all the APBS features you need shared libraries. The shared libraries (GCC 9.2) are not available to the end user unless they have our development environment. This affects Windows, MacOS, and Linux.

B. To build APBS statically, you don't get any python modules. Windows and Linux work but the MacOS build can't find the TABIPB library.

Until we get @ashermancinelli external build system working, we are kinda stuck going back and forth trying to get something working on all three platforms.

The Dockerfile.build and script I posted in #90 is probably the best option for all platforms until we get the build system fixed.

**Comment by sobolevnrm** [comment]
Perhaps we should offer both (A) and (B) for now and update the documentation to steer users in the right direction?

**Comment by ashermancinelli** [comment]
Did we previously go without SWIG-based python bindings in our prebuilt release?

**Comment by intendo** [comment]
Yes, we did use SWIG. I think pip is our ultimate goal for users to install APBS and all other packages.

**Comment by ashermancinelli** [comment]
> Until we get @ashermancinelli external build system working, we are kinda stuck 

Why are external builds blocking this? Prebuilt libs are just a convenience, I don't understand why they would block anything.

**Comment by intendo** [comment]
External builds are not blocking this. But there is not a good "fix" for the current build system on all platforms.
Windows build is broke - won't build with Python.
MacOSX build is broke - won't build statically.
Linux and Docker (based on Linux) will build statically with all APBS features.

The minimally prebuilt libs do not include the legacy compiler baggage and autoconf configurations that cause the Windows and MacOSX builds to fail trying to resolve linking all the libraries together.

**Comment by ashermancinelli** [comment]
This may have to branch off into another issue, but if the primary issue is usability we could just package everything into pip pretty easily. The setup.py could call the respective build script before installing the python bindings.

**Comment by sobolevnrm** [comment]
I'd like to have one last "stable" release before we start decomposition of the code into the Python-friendly version.

**Comment by ashermancinelli** [comment]
Sounds good. BTW the first pip-based release may well just build apbs for the user and install the python libs we already have, not much else.

**Comment by ashermancinelli** [comment]
Would this be closed by a release on pip?

**Comment by sobolevnrm** [comment]
Yes, I think so.
